docs: add enableReconnect and registerWalletStandard configuration options (#504)
reown-com/reown-docs: snippets/appkit/shared/multichain.mdx

</Tab>

</Tabs>

## Solana Adapter Configuration

The `SolanaAdapter` accepts configuration options to customize its behavior.

### registerWalletStandard

Controls whether WalletConnect is registered as an available WalletStandard wallet following the Solana Wallet Standard protocol. When enabled, users will see "WalletConnect" as a wallet choice alongside other Solana wallets like Phantom, Solflare, etc. when integrating wallet standard libraries

```ts
const solanaWeb3JsAdapter = new SolanaAdapter({

>   registerWalletStandard: false,

Reviewer: change to `true`

>   registerWalletStandard: false,

Reviewer: ```suggestion
  registerWalletStandard: true,
```

</Tab>

</Tabs>

## Solana Adapter Configuration

The `SolanaAdapter` accepts configuration options to customize its behavior.

### registerWalletStandard

> Enable or disable automatic registration of WalletConnect relay connector as a Wallet Standard wallet. Default is `true`.

Reviewer 2: What exactly does this mean? I don't really get it. 

Please explain it clearly while being concise. Also add a simple use-cases for this

</Tab>

</Tabs>

## Solana Adapter Configuration

The `SolanaAdapter` accepts configuration options to customize its behavior.

### registerWalletStandard

> Controls whether WalletConnect appears as an available wallet option in Solana wallet selection interfaces that follow the Wallet Standard protocol. When enabled (default), users will see "WalletConnect" as a wallet choice alongside other Solana wallets like Phantom, Solflare, etc.

Reviewer: ```suggestion
Controls whether WalletConnect is registered as an available WalletStandard wallet following the Solana Wallet Standard protocol. When enabled, users will see "WalletConnect" as a wallet choice alongside other Solana wallets like Phantom, Solflare, etc. when integrating wallet standard libraries
```
